testing/ios/IosUnitTests/IosUnitTests.xcodeproj/project.pbxproj

```diff
@@ -64,6 +64,7 @@
 		0D6AB6C922BB05E200EEE540 /* IosUnitTestsTests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = IosUnitTestsTests.xctest; sourceTree = BUILT_PRODUCTS_DIR; };
 		0D6AB6CF22BB05E200EEE540 /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		0D6AB73E22BD8F0200EEE540 /* FlutterEngineConfig.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = FlutterEngineConfig.xcconfig; sourceTree = "<group>"; };
+		68FF0A70255A3B4E00165B60 /* FlutterPlatformViewsTest.mm */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.objcpp; path = FlutterPlatformViewsTest.mm; sourceTree = "<group>"; };
```

Comment:
I found that this test file is not referenced by the project, so while I'm here, I added it to the project so it helps the debugging process easier.

Reply:
It isn't suppose to be in here.  That is linked in as part of ios_test_flutter_mrc https://github.com/flutter/engine/blob/5d4d06ebecb4f39d4a9e1b206072e3420a39275a/shell/platform/darwin/ios/BUILD.gn#L180

You'll get duplicate symbols if you do this.

Reply:
You still need to remove this.